From 85ba9ee8d95d144252a4dff5598859d85a2ad4c3 Mon Sep 17 00:00:00 2001
From: Michael Glembotzki <Michael.Glembotzki@iris-sensing.com>
Date: Thu, 10 Mar 2022 11:03:36 +0100
Subject: [PATCH] imx8mp-irma6r2.dts: Add support for the PCF8563TS RTC

[    2.187156] rtc rtc1: invalid alarm value: 2022-03-10T11:85:00
[    2.193394] rtc-pcf8563 2-0051: registered as rtc1

Signed-off-by: Michael Glembotzki <Michael.Glembotzki@iris-sensing.com>
Signed-off-by: Jasper Orschulko <jasper@fancydomain.eu>
---
 arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts b/arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts
index 4945eaaaf0b1..679baf7b326b 100644
--- a/arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts
+++ b/arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts
@@ -395,6 +395,12 @@ ldo5: LDO5 {
 			};
 		};
 	};
+
+	rtc: rtc@51 {
+		compatible = "nxp,pcf8563";
+		reg = <0x51>;
+		#clock-cells = <0>;
+	};
 };
 
 &i2c4 {
-- 
2.46.0

